README.md

```diff
@@ -85,6 +90,7 @@ None
   roles:
     - ansible-role-fluentd
   vars:
+    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% endif %}"
```

Comment:
`-p` is not necessarily needed. it is only when you want to install _local_ plug-ins *AND* `fluentd` behaves same on all distributions. the port should have patched the gem to include `/usr/local/etc/fluentd/plugin`.